ovirt_imageio/client/_options.py

```diff
@@ -64,6 +70,18 @@ class Parser:
             help=("Path to ovirt-engine CA certificate. If not set, read "
                   "from the specified config section"),
         ),
+        Option(
+            args=["--log-file"],
+            name="log_file",
+            help="If set, log to file.",
+        ),
+        Option(
+            args=["--log-level"],
+            name="log_level",
+            type=log_level,
+            default="info",
+            help="Logging level (default: info).",
```

Comment:
As an alternative, https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter allows to have the default value added automatically to the help messages.

Reply:
It does not work for us, since we must define all options *without* a default. If we use a default, after parsing arguements, all arguments have a default, so when we come to merge the config values we cannot detect an argument which was not set.